gnovm/Makefile

```diff
@@ -0,0 +1,71 @@
+.PHONY: help
+help:
+	@echo "Available make commands:"
+	@cat Makefile | grep '^[a-z]' | grep -v '=' | cut -d: -f1 | sort | sed 's/^/  /'
+
+rundep=go run -modfile ../misc/devdeps/go.mod
+
+.PHONY: build
+build:
+	go build -o build/gno ./cmd/gno
+
+.PHONY: install
+install:
+	go install ./cmd/gno
+
+.PHONY: clean
+clean:
+	rm -rf build
+
+########################################
+# Dev tools
+.PHONY: lint
+lint:
+	golangci-lint run --config ../.github/golangci.yml ./...
+
+.PHONY: fmt
+GOFMT_FLAGS ?= -w
+fmt:
+	$(rundep) mvdan.cc/gofumpt $(GOFMT_FLAGS) .
+	$(rundep) mvdan.cc/gofumpt $(GOFMT_FLAGS) `find pkg/stdlibs -name "*.gno"`
+
+########################################
+# Test suite
+.PHONY: test
+test: _test.cmd _test.pkg _test.gnolang
+	@echo "Full test suite finished."
```

Comment:
This is a lie (it may be printed on the top-level makefile, and it may not be entirely finished), I propose just to remove this echo altogether, on the principle of "no news is good news"?

Reply:
removed